src/libp2p_transport_tcp.erl

```diff
@@ -695,7 +695,7 @@ tcp_listen_addrs(Socket) ->
 
 reuseport0(TID, Type, {0,0,0,0}, 0) ->
     Cache = libp2p_swarm:cache(TID),
-    case libp2p_cache:lookup(Cache, {tcp_listen_addrs, Type}, []) of
+    case libp2p_cache:lookup(Cache, {tcp_local_listen_addrs, Type}, []) of
```

Comment:
This rename doesn’t fix anything right? Just makes it clearer what is being cached?

Reply:
yes